src/routes/RecipeDetails.js

```diff
@@ -34,6 +34,24 @@ class RecipeDetails extends React.Component {
             {__html: recipeDetail?.description}
           }>
         </div>
+        <h6 className="text-left text-secondary text-uppercase mb-4 mt-4">{recipe.title}</h6>
```

Comment:
There is a class for margin **top AND bottom** 😉  let's go for that one

Reply:
Also .. I see you need to use of a lot of `text-left` classes. You can avoid that. The default is `text-align: left;`
So I would either only add the class `text-center` where needed. Or if you want to use your previous approach, wrap the title, description, and image in a wrapper div with the class `text-center`. In any case. I would get rid of all the `text-left` classes